client/src/app/site/pages/meetings/modules/poll/components/chart/chart.component.ts

```diff
@@ -157,7 +157,12 @@ export class ChartComponent {
                 // removes undefined and null values
                 return chartDate.data.filter(data => !!data);
             });
-            if (!this.circleColors || inputChartData.length !== this.circleColors[0].backgroundColor.length) {
+            if (
+                !this.circleColors ||
+                ![inputChartData.length, inputChartData[0].data.length].includes(
+                    this.circleColors[0].backgroundColor.length
+                )
```

Comment:
I think this should be
```suggestion
                ![this.circleColors[0].backgroundColor.length, this.circleColors[0].hoverBackgroundColor.length].includes(
                    inputChartData.length
                )
```